docs/demo/deployments/xcluster/ovl/meridio/description.md

```diff
@@ -0,0 +1,204 @@
+
+The TG enables ECMP routing to allow installing multipath routes in case of multiple Gateways.
+
+#### Secondary Networks
+
+The required Secondary Networks (VLAN) are provided through NetworkAttachementDefinitions with static IPAM.
+```
+# example NADs used by Gateways
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: vlan-conf
+spec:
+  config: '{
+      "cniVersion": "0.3.1",
+      "plugins": [
+        {
+          "type": "vlan",
+          "capabilities": { "ips": true },
+          "master": "eth1",
+          "vlanId": 100,
+          "ipam": {
+            "type": "static"
+          }
+        }
+      ]
+    }'
+---
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: gwtg-conf
+spec:
+  config: '{
+      "cniVersion": "0.3.1",
+      "plugins": [
+        {
+          "type": "vlan",
+          "capabilities": { "ips": true },
+          "master": "eth2",
+          "vlanId": 3000,
+          "ipam": {
+            "type": "static"
+          }
+        }
+      ]
+    }'
+```
+
+The related IP addresses are configured through annotations in the manifests files.
+
+```
+# example Gateway network annotation
+      annotations:
+        k8s.v1.cni.cncf.io/networks: '[
+          { "name": "vlan-conf",
+            "ips": [ "169.254.100.254/24", "fe80::beef/64", "100:100::254/64" ],
+           "interface": "vlan0"
+          },
+         { "name": "gwtg-conf",
+           "ips": [ "11.0.0.1/24", "1111::1/100" ],
+           "interface": "tg"
+         }]'
+```
+
+The interface names are currently hardcoded.
+That is in the Gateway **vlan0** serves Frontends, while the interface facing the TG is called **tg**.
+Similarly in the Traffic Generator the interface **tg** is used to interact with the Gateways.
+
+Note: Should the interface names change, BIRD configuration or BIRD wrapper scripts must be also updated.
+
+Avoiding IP address collisions between the Gateways and connected Meridio Frontends is outside the scope of this description.
+But Network Service Mesh (NSM) the Meridio Frontends rely on have support to reserve IP addresses so that they do not get allocated.
+
+#### Helm
+
+Deploy two Gateways and a Traffic Generator with [helm](https://github.com/Nordix/Meridio/tree/master/docs/demo/deployments/xcluster/ovl/meridio/helm/gateway).
+
+Some of the parameters the chart allows to configure:
+- VLAN ID of the network between Merdio Frontends and Gateways (default=100)  
+  Note: Must be in sync with the value used by the Meridio Frontends to be connected with.
+- VLAN ID of the network between the TG and Gateways (default=3000)  
+- host node interfaces to be used as VLAN masters (default=eth0)  
+- Gateway IP addresses to be used on the VLAN interface facing Meridio Frontends  
+  (default; gw1IPv4=169.254.100.254/24, gw1IPv6_2=100:100::254/64, gw2IPv4=169.254.100.253/24, gw2IPv6_2=100:100::253/64)  
```

Comment:
for some reason 100:100::254/64 is not displayed correctly: https://github.com/Nordix/Meridio/blob/5f995a76b840400efacd75acda0937ad47cabe44/docs/demo/deployments/xcluster/ovl/meridio/description.md#helm